plugins/services/src/js/stores/SDKEndpointStore.js

```diff
@@ -0,0 +1,149 @@
+import PluginSDK from "PluginSDK";
+
+import { SERVER_ACTION } from "#SRC/js/constants/ActionTypes";
+import AppDispatcher from "#SRC/js/events/AppDispatcher";
+import GetSetBaseStore from "#SRC/js/stores/GetSetBaseStore";
+
+import {
+  REQUEST_SDK_ENDPOINTS_SUCCESS,
+  REQUEST_SDK_ENDPOINTS_ERROR,
+  REQUEST_SDK_ENDPOINT_SUCCESS,
+  REQUEST_SDK_ENDPOINT_ERROR
+} from "../constants/ActionTypes";
+
+import SDKEndpointActions from "../events/SDKEndpointActions";
+import SDKServiceEndpoint from "../structs/SDKServiceEndpoint";
```

Comment:
The `SDKServiceEndpoint` struct is missing. Please add the file or remove the import and the respective lines.

Reply:
Merged PR 02 and 03 together. Now this has the store, tests and the struct.

Reply:
Also kept the 2 PRs for now because both have comments, once one is merged the other can be deleted.